src/components/date-picker/date-picker.st.css

```diff
@@ -22,36 +22,50 @@
     dropdownWidth: 330px;
     dropdownArrowWidth: 11px;
     dropdownArrowHeight: 12px;
-    iconWidth: 36px;
+    iconSize: 16px;
+    iconOffset: 37px;
+    iconArrowOffset: 36px;
+}
+
+.root {
+    display: inline-block;
 }
 
 /* The DatePicker Input */
 .input {
     width: value(inputWidth);
+    height: value(inputHeight);
```

Comment:
Remove the width and height from the component. When you enter something in the reset style, it means that the user will have to override it. All you need to provide is a way to change it (no values)